packages/components/src/Typeahead/Typeahead.component.js

```diff
@@ -140,7 +140,7 @@ function Typeahead({ onToggle, icon, position, docked, ...rest }) {
 		...inputProps,
 		items: getItems(rest.items, rest.dataFeature),
 		itemProps: ({ itemIndex }) => ({
-			onMouseDown: event => event.preventDefault(),
+			onMouseDown: rest.onSelect,
```

Comment:
The `onClick` handler is needed for usage tracking reasons, if we handle `onMouseDown` before I'm not sure that the `onClick` will be fired.
Could you double-check, please? 

Related to https://github.com/Talend/ui/pull/2753